demos/adaptive_circuits_demo.ipynb

```diff
@@ -978,7 +986,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "params_doubles: [ 0.04758607 -0.09855445  0.09865958  0.05330061  0.05331783  0.22938123]\n"
```

Comment:
Hey @tzunghanjuang, it surprises me that the new output here shows all the same value in the list 🤔 Did something change/go wrong here?

Reply:
@tzunghanjuang did mention this in possible drawbacks

> I cannot reproduce some results (which are not related to jaxopt and optax) in demos. Might be the issues of environment setup and random seeds.

Maybe this is an example?